frame/balances/src/weights.rs

```diff
@@ -0,0 +1,107 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Weights for pallet_balances
+//! ./target/release/substrate benchmark --pallet pallet_balances --extrinsic * --repeat 20 --steps 50 --output ./frame/balances/src/weights.rs --template ./.maintain/frame-weight-template.hbs --execution=wasm --wasm-execution=compiled --header HEADER --db-cache 512 --chain dev
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.0
+//! DATE: 2020-10-26, STEPS: [50, ], REPEAT: 20, LOW RANGE: [], HIGH RANGE: []
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 512
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use sp_std::marker::PhantomData;
+
+pub trait WeightInfo {
+	fn transfer() -> Weight;
+	fn transfer_keep_alive() -> Weight;
+	fn set_balance_creating() -> Weight;
+	fn set_balance_killing() -> Weight;
+	fn force_transfer() -> Weight;
+	
```

Comment:
should be better to remove ths whitespace with `~` somehow

Reply:
Yeah, i am still working on that, couldnt find the right combination of tricks to remove whitespace while keeping this formatted right in all situations

Reply:
Seems there is a bug in handlebars.rs which is ultimately causing this: https://github.com/sunng87/handlebars-rust/issues/258